.github/workflows/cuda.yml

```diff
@@ -0,0 +1,71 @@
+name: CUDA GitHub Actions
+
+on:
+  pull_request_review_comment:
+    types: [created]
```

Comment:
Probably simple `issue_comment` will be easier, but it requires workflow config in the `master` branch, so I cannot test it right now in this PR.
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#issue_comment

Reply:
oh interesting. I think it's ok to leave it like this if it works!